### ISSUE TITLE
How to contribute in such a way that you can see Organizations in your profile ?

### ISSUE THREAD

**Issue by mehul-m-prajapati** [OpenSourceHelpCommunity/Questions#1]
Hi,

I can see different Organization name in some user's profiles.
How can I become member of any organizations ?

**Comment by lakshaykalbhor** [comment]
Hey, I don't know if I'm the best person to answer since I'm a beginner, but I was recently invited to my first organisation. All I did was I went through their projects and found the one's I could contribute to, then I looked through the code to see if there was something I could optimise, or if I could add a feature. eg : In one of their projects they were using a python module - mechanize to extract search results from Google, so I thought of improving it by using simple http requests and beautifulsoup scraping. This removed one dependency and made the project working for python 3 too (mechanize wasn't available on python 3). 

After I sent a pull request, the admin found it useful and merged it into the main project. Then I was invited to the organisation.
Basically, solve some issue or add some feature and they'll probably take you in depending on the organisation.

**Comment by sara-02** [comment]
@mehul-m-prajapati.
Some ways in which you can get added to organisations are:
1. Regularly contribute to that organisation.
2. Be one of the founding members of a new org
3. Join an organisation as a part of GSoC or other such programs.
4.  Become a core contributor or reviewer of an org or a project of the org.

**Comment by mehul-m-prajapati** [comment]
Thanks @sara-02 and @lakshaykalbhor
